features/bootstrap/FeatureContext.php

```diff
@@ -394,21 +394,23 @@ private function getExpectedOutput(PyStringNode $expectedText)
      */
     public function itShouldFail($success)
     {
-        $message = '';
 
-        if ('fail' === $success) {
-            if (0 === $this->getExitCode()) {
-                $message = 'Actual output:' . PHP_EOL . PHP_EOL . $this->getOutput();
-            }
-
-            Assert::assertNotEquals(0, $this->getExitCode(), $message);
-        } else {
-            if (0 !== $this->getExitCode()) {
-                $message = 'Actual output:' . PHP_EOL . PHP_EOL . $this->getOutput();
-            }
-
-            Assert::assertEquals(0, $this->getExitCode(), $message);
-        }
+        $is_correct = match($success) {
+            'fail' => 0 !== $this->getExitCode(),
+            'pass' => 0 === $this->getExitCode(),
+       };
+     
+       if ($is_correct) {
+        return;
+       }
+    
+       throw new UnexpectedValueException(
+          'Expected previous command to '.strtoupper($success).' but got exit code '.$this->getExitCode().PHP_EOL
+          .PHP_EOL
+          .'##### Actual output #####'.PHP_EOL
+          .'> '.str_replace(PHP_EOL, PHP_EOL . '> ', $this->getOutput()).PHP_EOL
+          .'##### End of output #####'
+        );
```

Comment:
Looks good, but please fix code style - indents are 4 spaces and there should be a space around concat (period) operator:
```php
        $is_correct = match ($success) {
            'fail' => 0 !== $this->getExitCode(),
            'pass' => 0 === $this->getExitCode(),
        };

        if ($is_correct) {
            return;
        }

        throw new UnexpectedValueException(
            'Expected previous command to ' . strtoupper($success) . ' but got exit code ' . $this->getExitCode() . PHP_EOL
            . PHP_EOL
            . '##### Actual output #####' . PHP_EOL
            . '> ' . str_replace(PHP_EOL, PHP_EOL . '> ', $this->getOutput()) . PHP_EOL
            . '##### End of output #####'
        );
```

Reply:
Arg, so accustomed to use prettier-php that fixes automatically all this boring code style stuff 😄

Reply:
We are hoping to have a coding standard tool available here soon 😉